vue-frontend/src/components/home-new/PortfolioSection.vue

```diff
@@ -0,0 +1,259 @@
+      >
+        <div
+          v-if="activeIndex == portfolio.id"
+          :class="[
+            portfolio.classes,
+            activeIndex == 3
+              ? 'lg:tw--mt-[1.5rem] tw-py-[10px] lg:tw-pb-[35px] xl:tw-pb-2  tw-scale-[1.14] lg:tw-scale-[1.16] xl: tw-scale-[1.14] '
+              : activeIndex == 6
+              ? 'lg:tw--mt-[1.5rem] tw-pb-[45px] lg:tw-py-[45px]   tw-scale-110'
+              : activeIndex == 8
+              ? 'tw-h-full '
+              : activeIndex == 7
+              ? 'tw-px-[30px] xl:tw-px-[50px] tw-py-[25px] '
+              : 'tw-py-[10px] md:tw-py-[25px] 2xl:tw-py-[25px] tw-px-[10px] md:tw-px-[15px] 2xl:tw-px-[50px] 2xl:tw-pb-[50px]',
+          ]"
+          class="tw-absolute tw-z-[1] tw-shadow-[4px_0px_20px_5px_rgba(61,61,61,0.25)] tw-scale-125 tw-transition-all tw-duration-200"
+        >
+          <div
+            :class="[
+              activeIndex == 8
+                ? 'tw-absolute tw-p-4 tw-top-[15px] tw-left-[40px] lg:tw-left-[80px]  '
+                : '',
+            ]"
+            class="tw-pb-1 xl:tw-pb-5 tw-text-[1.375rem] tw-leading-[2rem] lg:tw-text-[2rem] lg:tw-leading-[2rem] xl:tw-text-[2.5rem] xl:tw-leading-[2rem] tw-text-center tw-font-bold tw-text-black-core/[0.87]"
+          >
+            {{ portfolio.title }}
+          </div>
+
+          <img
+            :class="[
+              activeIndex == 3 || activeIndex == 6
+                ? 'tw-px-[20px]'
+                : activeIndex == 1 ||
+                  activeIndex == 2 ||
+                  activeIndex == 4 ||
+                  activeIndex == 5 ||
+                  activeIndex == 7
+                ? 'tw-px-[50px] '
+                : '',
+            ]"
+            :src="portfolio.image[3]"
+            :srcset="`${portfolio.image[0]} ${portfolio.imageRenderSize[0]}, ${portfolio.image[1]} ${portfolio.imageRenderSize[1]}, ${portfolio.image[2]} ${portfolio.imageRenderSize[2]}, ${portfolio.image[3]} ${portfolio.imageRenderSize[3]}`"
+            sizes="(min-width: 992px) 45vw, 100vw"
+            class="tw-w-full tw-h-full tw-object-cover"
+            :alt="portfolio.title + `-image`"
+          />
+          <aspect-ratio
+            :class="[
+              portfolio.id == 2
+                ? 'tw-absolute  tw-w-[18.66667%]  md:tw-w-[23.777%] lg:tw-w-[28.666%] xl:tw-w-[30.666%] tw-h-full tw--top-[75px] sm:tw-top-[52px] md:tw--top-[131.5px] lg:tw--top-[219px] xl:tw--top-[285px]  tw-left-[52px] md:tw-left-[55px] lg:tw-left-[58px] xl:tw-left-[59.4px]'
+                : portfolio.id == 5
+                ? 'tw-absolute tw-w-[18.66667%]  md:tw-w-[17.6667%] lg:tw-w-[19.666%] xl:tw-w-[20.666%] tw-h-full tw--top-[75px] sm:tw-top-[52px] md:tw--top-[124px] lg:tw--top-[193px] xl:tw--top-[244px] 2xl:tw--top-[256px] tw-left-[52px] md:tw-left-[126px] lg:tw-left-[169px] xl:tw-left-[200px] 2xl:tw-left-[207px] '
+                : '',
+            ]"
+            class="tw-hidden md:tw-block"
```

Comment:
Why `hidden` in mobile devices?

Reply:
As i have given manually  absolute position to video , so I am unable to fix video in sm devices .